examples/js/exporters/GLTFExporter.js

```diff
@@ -647,20 +648,25 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
-
-			if ( ! ( material instanceof THREE.MeshStandardMaterial ) ) {
-
-				console.warn( 'GLTFExporter: Currently just THREE.MeshStandardMaterial is supported. Material conversion may lose information.' );
-
-			}
-
 			// @QUESTION Should we avoid including any attribute that has the default value?
 			var gltfMaterial = {
 
 				pbrMetallicRoughness: {}
 
 			};
 
+			if ( material instanceof THREE.MeshBasicMaterial ) {
```

Comment:
Off topic: 

@mrdoob If I'm right, Three.js recommends to use `object.is*` rather than `instanceof` for Three.js object, correct? If so, I'm gonna make PR to replace them because bunch of `instanceof` are used in `GLTFExporter`,

Reply:
Updated 👍